service-springboot-rest/pom.xml

```diff
@@ -46,6 +48,16 @@
             <groupId>org.keycloak</groupId>
             <artifactId>keycloak-spring-boot-starter</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
```

Comment:
Why are these dependencies needed? It seems like this may indicate an issue if they suddenly are required, but wasn't before.

Reply:
Those are needed by the REST client, which is responsible for uploading the realm.

Reply:
What REST client? Keycloak admin client? Applications shouldn't need to change their dependencies to use the admin client, so that may be an actual issue if that's the case